sdk/servicebus/azure-servicebus/azure/servicebus/_common/_configuration.py

```diff
@@ -0,0 +1,30 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from typing import Optional, Dict, Any
+
+from uamqp.constants import TransportType
+
+
+class Configuration(object):  # pylint:disable=too-many-instance-attributes
```

Comment:
Is this class and it's properties used entirely internally?

Reply:
yes, internal use only